scripts/intfstat

```diff
@@ -274,63 +275,34 @@ def main():
     delete_saved_stats = args.delete
     delete_all_stats = args.delete_all
     use_json = args.json
-    tag_name = args.tag if args.tag else ""
-    uid = str(os.getuid())
+    tag_name = args.tag
     wait_time_in_seconds = args.period
     interface_name = args.interface if args.interface else ""
 
-    # fancy filename with dashes: uid-tag / uid etc
-    filename_components = [uid, tag_name]
+    cnstat_file = "intfstat"
 
-    cnstat_file = "-".join(filter(None, filename_components))
+    cache = UserCache(tag=tag_name)
```

Comment:
Suggest creating tagged cache when `tag_name is not none`. Otherwise, we will write the same cache twice, as the   `cnstat_fqn_general_file` and `cnstat_fqn_file` are pointing to the same file.

Reply:
If ```tag_name is not None``` then indeed ```cnstat_fqn_general_file``` and ```cnstat_fqn_file``` are pointing to the same file. However, ```cnstat_fqn_general_file``` is only used here https://github.com/Azure/sonic-utilities/pull/2232/files#diff-e87da8afcfade426497bf52e7c166430974dd0b803d4a4af76cb0f0a7b09596fR306 after a check for ```tag_name is not None```.

Reply:
Make sense. Thanks